index.html

```diff
@@ -0,0 +1,145 @@
+            <h2>Activities</h2>
+
+            <ul class="program-list">
+                <li class="program-item">
+                    <div class="act-title">
+                        <i class="fa-solid fa-person-hiking"></i>
+                        <h3>Trekking</h3>
+                    </div>
+                    <p>Explore majestic trails, immerse yourself in nature's wonders, and let adventure ignite your soul.</p>
+                </li>
+                <li class="program-item">
+                    <div class="act-title">
+                        <i class="fa-solid fa-bed"></i>
+                        <h3>Lodging</h3>
+                    </div>
+                    <p>Unwind and indulge in pure bliss, where comfort meets tranquility, your perfect lodging escape.</p>
+                </li>
+                <li class="program-item">
+                    <div class="act-title">
+                        <i class="fa-sharp fa-solid fa-utensils"></i>
+                        <h3>Gastronomy</h3>
+                    </div>
+                    <p>Savor exquisite flavors, where culinary artistry meets your palate, an epicurean delight awaits.</p>
+                </li>
+                <li class="program-item">
+                    <div class="act-title">
+                        <i class="fa-solid fa-wine-glass"></i>
+                        <h3>Wine Cellar</h3>
+                    </div>
+                    <p>Elevate your senses with wines that reach new heights, where excellence and altitude intertwine.</p>
+                </li>
+                <li class="program-item">
+                    <div class="act-title">
+                        <i class="fa-solid fa-road"></i>
+                        <h3>Guidenes</h3>
+                    </div>
+                    <p>Embark on immersive journeys, unravel hidden gems, and create unforgettable memories through our guided tours</p>
+                </li>
+            </ul>
+            <button class="act-btn"> See all activities</button>
+        </div>
+        
+    </section>
+    <!-- Wine Cellar-->
+    <section class="w-cellars">
+        <h2>Wine Cellars</h2>
+        <div class="wc-container"></div>
+    </section> 
+    <!-- Sponsor -->
+    <section class="sponsor">
+        <h2>Sponsors</h2>
+        <ul class="sponsor-list">
+            <li class="sponsor-item"><img src="Images/Home/sponsor/salta-rv.png" alt=""></li>
+            <li class="sponsor-item"><img src="Images/Home/sponsor/uma-travel.png" alt=""></li>
+            <li class="sponsor-item"><img src="Images/Home/sponsor/majo-viajes.png" alt=""></li>
+        </ul>
+    </section>
+    <!-- Footer -->
+    <footer>
+        <div class="footer-info">
+            <a href="">
+                <img class="logo" src="Images/Home/Logo_Cafayate.png" alt="">
+            </a>
+            <p class="info">
+                © Copyright 2023 - Cappa Systems - All rights reserved
+            </p>
+        </div>
+    </footer>
```

Comment:
- [ ] Please 🙏🏻  adhere to the project requirement according to which you can make little changes or adjustments of your own choice but can't change it completely but as I can see that your design looks different than actual, which makes your design `inconsistent` and looks unprofessional. Please have a look at the below-attached image and table. Kindly also take a look at the `about.html` page's footer part because your about page footer is also not matching with the `index` HTML page. Both page's footer should be identical visaully.

### Requirement
![image](https://github.com/ulises2607/cayafate-project/assets/66476329/cd029716-65ab-45b5-8e7d-6886f05d48af)


### Desktop Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![image](https://github.com/ulises2607/cayafate-project/assets/66476329/d4082a25-d456-4b6b-bb10-973144b3fc4d)

</td>
<td>

![Screenshot from 2023-07-08 15-54-39](https://github.com/ulises2607/cayafate-project/assets/66476329/11fa9a47-a363-4194-9306-8c87a5988952)

</td>
</tr>
</table>

### Footer Design Comparison (Both should be identical)
<table>
<tr>
<th>About Page Footer</th>
<th>Index Page Footer</th>
</tr>
<tr>
<td>

![image](https://github.com/ulises2607/cayafate-project/assets/66476329/7b7eabfc-335c-4f52-a057-84cb774e5a03)

</td>
<td>

![Screenshot from 2023-07-08 15-54-39](https://github.com/ulises2607/cayafate-project/assets/66476329/11fa9a47-a363-4194-9306-8c87a5988952)

</td>
</tr>
</table>